errbot/backends/slack.py

```diff
@@ -504,7 +504,7 @@ def _message_event_handler(self, event):
             if subtype == "bot_message":
                 msg.frm = SlackBot(
                     self.sc,
-                    bot_id=event['bot_id'],
+                    bot_id=event.get('bot_id'),
```

Comment:
I'm a little bit worried what kind of effects this change could have where `bot_id` ends up being `None` in cases, but let's see how it goes. :smile: 

That said, if this is changed here, the same should be applied to the initialization of `SlackRoomBot` a little further down (line 519 currently).

Reply:
Made that change.